packaging/debian/postinst

```diff
@@ -52,20 +51,35 @@ case "$1" in
 	# TODO: should not download binaries if it's only a reconfigure
 	echo "Downloading binaries..."
 	for binary in "storagenode" "storagenode-updater"; do
-	    wget -O /tmp/${binary}.zip ${BINARIES_SERVER}/${binary}_linux_amd64.zip
-	    unzip /tmp/${binary}.zip -d /var/storagenode/
-	    mv /var/storagenode/${binary}_linux_amd64 /var/storagenode/${binary}
-	    # so that we're in PATH easily
-	    ln -s /var/storagenode/${binary} /usr/bin/${binary}
+	    wget -O - ${BINARIES_SERVER}/${binary}_linux_amd64.zip | gunzip > /var/lib/storagenode/${binary}
```

Comment:
If the binary is really a ZIP file, `gunzip` will not be able to decompress it. `gunzip` works on `gzip`ped files, which usually have a `.gz` or `.tgz` extension.